meilisearch/task.py

```diff
@@ -67,6 +68,37 @@ def get_task(config: Config, uid: int) -> dict[str, Any]:
     return http.get(f"{config.paths.task}/{uid}")
 
 
+def cancel_tasks(config: Config, parameters: dict[str, Any]) -> TaskInfo:
+    """Cancel a list of enqueued or processing tasks.
+
+    Parameters
+    ----------
+    config:
+        Config object containing permission and location of Meilisearch.
+    parameters (optional):
+        parameters accepted by the cancel tasks https://docs.meilisearch.com/reference/api/tasks.html#cancel-task.
+
+    Returns
+    -------
+    task_info:
+        TaskInfo instance containing information about a task to track the progress of an asynchronous process.
+        https://docs.meilisearch.com/reference/api/tasks.html#get-one-task
+
+    Raises
+    ------
+    MeiliSearchApiError
+        An error containing details about why Meilisearch can't process your request. Meilisearch error codes are described here: https://docs.meilisearch.com/errors/#meilisearch-errors
+    """
+    http = HttpRequests(config)
+    if parameters is None:
+        parameters = {}
+    for param in parameters:
+        if isinstance(parameters[param], list):
+            parameters[param] = ",".join(parameters[param])
+    response = http.post(f"{config.paths.task}/cancel?{parse.urlencode(parameters)}")
+    return TaskInfo(**response)
```

Comment:
Just ouf ot curiosity, are this pointers? And if yes, why do we need to use it?

Reply:
It is dictionary unpacking. It's shorthand for `TaskInfo(index_uid=response.get("indexUid"), status=response.get("status"), ...`